nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/azure/eventhub/GetAzureEventHub.java

```diff
@@ -64,8 +64,12 @@
 import org.apache.nifi.processors.azure.eventhub.utils.AzureEventHubUtils;
 
 @Tags({"azure", "microsoft", "cloud", "eventhub", "events", "streaming", "streams"})
-@CapabilityDescription("Receives messages from Microsoft Azure Event Hubs, writing the contents of the Azure message to the content of the FlowFile. "
-        + "Note: Please be aware that this processor creates a thread pool of 4 threads for Event Hub Client. They will be extra threads other than the concurrent tasks scheduled for this processor.")
+@CapabilityDescription("Receives messages from Microsoft Azure Event Hubs, writing the contents of the message to the content of the FlowFile. "
```

Comment:
```suggestion
@CapabilityDescription("ConsumeAzureEventHub offers the recommended approach to receiving messages from Azure Event Hubs. GetAzureEventHub receives messages from Microsoft Azure Event Hubs without reliable checkpoint tracking. "
```

Reply:
I've swapped the two sentences because it felt strange that the first sentence is not about GetAzureEventHub in the documentation of the said processor.